lib/node_modules/@stdlib/stats/strided/ztest2/README.md

```diff
@@ -0,0 +1,238 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+<!-- lint disable max-heading-length -->
+
+# ztest2
+
+> Compute a two-sample Z-test for a strided array.
```

Comment:
```suggestion
> Compute a two-sample Z-test.
```

A two-sample Z-test operates on more on than array. I think it is fine to leave out `for a strided array` in this case here and elsewhere throughout this PR.